pom.xml

```diff
@@ -271,16 +277,6 @@
 				<groupId>org.apache.kafka</groupId>
 				<artifactId>kafka_2.10</artifactId>
 				<version>0.8.2.2</version>
-				<exclusions>
```

Comment:
"In the end, we choose at packaging time which logger we inject"

As long as there is hard code to an implementation binding, that is not true.  You *cannot* bind to anything other than log4j (I tried to keep the logback config, but could not).

Because of this, it really doesn't make sense to pretend we can bind to anything else.  Also, I'm not sure that [log4j calls] -> slf4j:slf4j-log4j12 -> slf4j:log4j-over-slf4j is an identity operation.

Which calls my attention to the fact that I bound to slf4j:slf4j-log4j12 in the jmxtrans:jmxtrans pom, instead of slf4j:log4j-over-slf4j... so why does this work?!

*SIGH*